A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,345 @@
+class A32NX_TransitionAltitude {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.checkStart = true;
+        this.checkStartSec = 0;
+        this.testAirport = "";
+        this.trySendRequest();
+    }
+
+    update(_deltaTime, _core) {
+        if (this.offline === true) {
+            this.offlineTransAlt();
+        }
+        this.tryCheckAPI(_deltaTime);
+    }
+
+    async trySendRequest() {
+        await NXApi.getAirport("KLAX")
+            .then((data) => {
+                this.testAirport = data.transAlt;
+            });
+    }
+
+    tryCheckAPI(_deltaTime) {
```

Comment:
**nitpick**: shouldn't start with `_`.
**thought**: this function is called `tryCheckAPI` but doesn't do anything with any API. That's odd.

Reply:
Because request api checking will work on init. 
on init, send KLAX TA request
On update, check KLAX TA is valid(only 30 sec)

Reply:
The name is `tryCheckAPI`, but it doesn't do anything with an API. Then the name can be improved right? 😀

Reply:
sure change to other name